new-library.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
```

Comment:
Sure, I saw those issues but figured it was best to stay consistent initially. Let me have a stab at converting this stuff to node.